tests/test_quotas.py

```diff
@@ -0,0 +1,77 @@
+import os
```

Comment:
I am not quite clear about this script. Is it for running a test for debugging once the quotas are created or just a unit test?